widget/balance/src/main/java/com/ivy/widget/balance/WalletBalanceWidget.kt

```diff
@@ -28,10 +29,19 @@ import kotlinx.coroutines.MainScope
 import kotlinx.coroutines.launch
 import javax.inject.Inject
 
+const val THOUSAND = 1000
+
 class WalletBalanceWidget(
     private val getAppStarter: () -> AppStarter,
 ) : GlanceAppWidget() {
-
+    private fun remember(balance: String): String {
```

Comment:
Hey, you need to use Compose `remember` your remember doesn't memoize

Reply:
@DripYeager to give you a hint:
```kotlin
val balanceFormatted = remember(balance) {
   // all of your code here...
}
```

Reply:
thanks for the help, i have updated the code let me if still there is any issue